andrew-oyanguren/typescript-portfolio
# PR-35 Experience Page Large Devices

### Summary

This work improves the Experience Page UI on large devices, as well as various CSS improvements throughout the app.

### Changes

- New Grid container on JrEngineer UI component.
- New `experienceBarConstants` and `deviceWidthConstants`.
- **ExperienceBar** helpers and test.

### Testing Guide

1. `npm run dev`
2. Navigate to the Experience page.
3. Open dev tools check out different views! (mainly focused on large devices)
**WebDev UI:**
<img width="937" alt="Screen Shot 2024-01-02 at 12 23 34 PM" src="https://github.com/andrew-oyanguren/typescript-portfolio/assets/69892684/1369b84a-bcdd-4821-9157-014fd73f9960">
**JrEng UI:**
<img width="925" alt="Screen Shot 2024-01-02 at 12 23 48 PM" src="https://github.com/andrew-oyanguren/typescript-portfolio/assets/69892684/83276f8d-9b67-4ae5-8ce6-609182dd4b49">
**SofwareEng UI:**
<img width="927" alt="Screen Shot 2024-01-02 at 12 24 05 PM" src="https://github.com/andrew-oyanguren/typescript-portfolio/assets/69892684/9709809f-b8d5-494c-bfc5-2c626ab5d2c1">
